atlasdb-api/src/main/java/com/palantir/atlasdb/spi/AtlasDbFactory.java

```diff
@@ -89,6 +90,8 @@ TimestampService createTimestampService(
             Optional<TableReference> timestampTable,
             boolean initializeAsync);
 
+    TimestampManagementService getTimestampManagementService(TimestampService timestampService);
```

Comment:
So this was actually intentional - since you're not creating a new access point, but instead converting the existing one. I.e. get timestamp management service from this given timestamp service, rather than create a new one.

Happy to change if you think `create` is a better name anyway,